libs/chrome-ext/src/lib/style.css

```diff
@@ -37,7 +41,11 @@ button {
   border-radius: 4px;
   cursor: pointer;
 }
-
+button.secondary {
+  border: 1px solid #ff0066;
+  color: #ff0066;
+  background-color: #fff;
```

Comment:
I know you don't have access to the CSS variables we have in Sistema, but I would still recommend you to choose the colors in our palette.
https://nuclia.github.io/frontend/palette